client/lib/feed-stream-store/paged-stream.js

```diff
@@ -249,16 +249,19 @@ export default class PagedStream {
 	}
 
 	dismissPostsFromSite( blogId ) {
```

Comment:
Doesn't the argument here need to be the `postId`?
While for the API it dismisses a site, for the client we are dismissing a post. 
Otherwise if a stream has many recs from the same blogId we wouldn't know which to actually dismiss